packages/sql-pg/src/PgClient.ts

```diff
@@ -169,7 +169,11 @@ export const make = (
             if (err) {
               resume(Effect.fail(new SqlError({ cause: err, message: "Failed to execute statement" })))
             } else {
-              resume(Effect.succeed(result.rows))
+              // Multi-statement queries return an array of results
+              const rows = Array.isArray(result)
+                ? result.flatMap((r) => r.rows ?? [])
```

Comment:
```suggestion
                ? result.map((r) => r.rows ?? [])
```

I think there should be one set per query, the same as the mysql driver